CorsixTH/Lua/languages/english.lua

```diff
@@ -874,6 +874,28 @@ map_editor_window = {
   }
 }
 
+fax = {
+vip_visit_result = {
+  ordered_remarks = {
```

Comment:
@TheCycoONE could this pose a potential copyright risk? Or is this kind of use ok?

Reply:
IANAL. We've been tredding carefully to date. I think to make corrections to an insubstantial part of the piece and requiring the purchase of the original puts us in the clear. Better if we wrote our own blurb though.